src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.domain.FruitTransaction;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(FruitTransaction fruitTransaction) {
+        Storage.getFruits().add(fruitTransaction);
+    }
+}
```

Comment:
is make sense for one method, maybe better write it in storage or write here all crud operations here